tests/functional/z_cluster/test_multiple_mds.py

```diff
@@ -0,0 +1,149 @@
+        2. Verify active and standby-replay mds count is same.
+        3. Perform node replacement on a newly added mds pod running node.
+        4. Make sure all the active mds pods come to active state.
+
+        """
+        original_active_count_cephfilesystem = get_active_mds_count_cephfilesystem()
+
+        # Scale up active mds pods from 1 to 3 sequentially.
+        new_active_mds_count = original_active_count_cephfilesystem + 2
+        adjust_active_mds_count_storagecluster(new_active_mds_count)
+
+        # Verify active and standby-replay mds counts.
+        verify_active_and_standby_mds_count(new_active_mds_count)
+
+        # Replace node
+        active_mds_pods = get_active_mds_pods()
+        active_mds_pod = random.choice(active_mds_pods)
+        active_mds_node_name = active_mds_pod.data["spec"].get("nodeName")
+        log.info(f"Replacing active mds node : {active_mds_node_name}")
+        delete_and_create_osd_node(active_mds_node_name)
+
+        # Perform cluster and Ceph health checks
+        self.sanity_helpers.health_check(tries=120)
+
+        # Verify active and standby-replay mds counts after node replacement
+        verify_active_and_standby_mds_count(new_active_mds_count)
+
+    def test_node_drain_and_fault_tolerance_for_multiple_mds(self):
+        """
+        1. Trigger the scale-up process to add new pods.
+        2. Drain active mds pod running node.
+        3. Verify active and standby-replay mds count is same.
+        4. Fail one active mds pod [out of two] and standby pod changes to active.
+
+        """
+
+        original_active_count_cephfilesystem = get_active_mds_count_cephfilesystem()
+
+        # Scale up active mds pods from 1 to 2.
+        new_active_mds_count = original_active_count_cephfilesystem + 1
+        adjust_active_mds_count_storagecluster(new_active_mds_count)
+
+        # Get active mds node name
+        active_mds_pods = get_active_mds_pods()
+        active_mds_pod = random.choice(active_mds_pods)
+        active_mds_pod_name = active_mds_pod.name
+        selected_pod_obj = pod.get_pod_obj(
+            name=active_mds_pod_name, namespace=config.ENV_DATA["cluster_namespace"]
+        )
+        active_mds_node_name = selected_pod_obj.data["spec"].get("nodeName")
+
+        # Drain active mds pod running node
+        node.drain_nodes([active_mds_node_name])
+
+        # Make the node schedulable again
+        node.schedule_nodes([active_mds_node_name])
+
+        # Perform cluster and Ceph health checks
+        self.sanity_helpers.health_check(tries=120)
+
+        # Verify active and standby-replay mds counts.
+        verify_active_and_standby_mds_count(new_active_mds_count)
+
+        # Fail one active mds pod [out of two]
+        rand = random.randint(0, 1)
+        ct_pod = pod.get_ceph_tools_pod()
+        ct_pod.exec_ceph_cmd(f"ceph mds fail {rand}")
+
+        # Verify active and standby-replay mds counts is still same.
```

Comment:
we are failing one active mds. would the count change in verify_active_and_standby_mds_count()?